remote/src/main/scala/org/apache/pekko/remote/transport/netty/NettyTransport.scala

```diff
@@ -481,7 +481,7 @@ class NettyTransport(val settings: NettyTransportSettings, val system: ExtendedA
     bootstrap
   }
 
-  private val inboundBootstrap: Bootstrap = {
+  private val inboundBootstrap: ServerBootstrap = {
```

Comment:
Need a mima filter for this

Reply:
I've fixed the issue - it was an import that needed removing, not a mima issue

The link validation failure - I have a fix in #640